README.md

```diff
@@ -376,6 +376,28 @@ cred.updateAnchor().then(() => {
 })
 ```
 
+#### Granting the Credential Usage(for single user) from the owner
+Since the Verifiable Credential is an immutable structure that is anchored on a immutable database(blockchain),
+someone can ask. "What if someone else gets a copy of the VC and tries to use that as if is the owner of it latter?"
```

Comment:
Fix english:

What if someone else gets AN copy of the VC and tries to use it LATTER posing as the owner